### PR TITLE
Sales: Add unit test for validator model class

### PR DESCRIPTION
**Pull request by dmytro-ch**
### Description
This PR adds missing unit test for:
``\Magento\Sales\Model\Validator``

### Fixed Issues
N/A

### Manual testing scenarios
N/A

### Contribution checklist
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me {$VERSION} instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation

File: app/code/Magento/Sales/Test/Unit/Model/ValidatorTest.php [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+            ->with($validatorName, $validatorArguments)->willReturn($this->validatorMock);
+        $this->validatorMock->expects($this->once())->method('validate')->with($this->entityMock)
+            ->willReturn($messages);
+        $this->validatorResultMock->expects($this->once())->method('addMessage')->with($message);
+
+        $expected = $this->validatorResultMock;
+        $actual = $this->validator->validate($this->entityMock, $validators, $context);
+        $this->assertEquals($expected, $actual);
+    }
+
```
Comment (rogyar) [inline app/code/Magento/Sales/Test/Unit/Model/ValidatorTest.php:115]: Please, remove the extra empty line

**Review by rogyar — CHANGES_REQUESTED** [review]

File: app/code/Magento/Sales/Test/Unit/Model/ValidatorTest.php (hunk 1 above)
Comment (dmytro-ch) [reply app/code/Magento/Sales/Test/Unit/Model/ValidatorTest.php:115]: @rogyar, thank you! I've fixed it.

**Comment by magento-engcom-team** [comment]
Hi @dmytro-ch. Thank you for your contribution. 
We will aim to release these changes as part of 2.2.7. 
Please check the release notes for final confirmation.

Please, consider to port this solution to 2.3 release line.
You may use [Porting tool](https://github.com/magento/magento2/wiki/Simple-way-to-port-your-solutions-automatically) to port commits automatically.

**Pull request merged**